test/config.yml

```diff
@@ -3,4 +3,6 @@ memory: [no-reuse-impls, no-reuse, reuse]
 time-limit: 120
 memory-limit: 450
 flags: [--unroll=2]
+checkbpl: []
```

Comment:
The `regtest.py` script requires that fields like `flags` and `checkbpl` that can be appended to by nested `config.yml` files be initialized in the root `test/config.yml`. Otherwise, e.g., this line would fail:

https://github.com/smackers/smack/blob/master/test/regtest.py#L80